twiggy/tests/tests.rs

```diff
@@ -47,11 +47,13 @@ macro_rules! test {
             let expected = slurp(expected_path).unwrap_or(vec![]);
             let expected = String::from_utf8_lossy(&expected);
             let expected = expected.trim();
+            let expected_lines = expected.lines().collect::<Vec<&str>>();
 
             let actual = String::from_utf8_lossy(&output.stdout);
             let actual = actual.trim();
+            let actual_lines = actual.lines().collect::<Vec<&str>>();
 
-            if actual != expected {
+            if actual_lines != expected_lines {
```

Comment:
I think we should also start using

```rust
diff::slice(&expected_lines, &actual_lines)
```

just below this as well.

Reply:
But only for performance reasons, right @fitzgen? Since `diff::lines()` already ignores the different line endings...

Reply:
I'm also thinking about if `diff::Result::Both()` hunks should be really printed... Since they make it sometimes hard to find the actual difference in large outputs

Reply:
And is there an easy way to color those `diff` outputs btw? :D